enarx-keep-sgx/src/main.rs

```diff
@@ -70,8 +71,16 @@ fn main() {
         }
 
         // Complete the process.
-        builder.done().expect("Unable to initialize enclave")
+        (
+            builder.done().expect("Unable to initialize enclave"),
+            code.entry,
+        )
     };
 
-    enclave.enter(Leaf::Enter).unwrap();
+    loop {
+        match enclave.enter(entry, 0, 0, Leaf::Enter, 0, 0) {
+            Err(Some(ei)) if ei.trap == Exception::InvalidOpcode => (),
```

Comment:
I'm not sure I follow this branch. Is this just ignoring invalid opcodes?

Reply:
Fixed via comment explaining the loop.